sys/net/gnrc/application_layer/uhcpc/gnrc_uhcpc.c

```diff
@@ -54,6 +57,25 @@ static void set_interface_roles(void)
 
 static ipv6_addr_t _prefix;
 
+#ifdef MODULE_GNRC_SIXLOWPAN_CTX
+#define SIXLO_CTX_LTIME (60)
+
+static void _update_prefix(ipv6_addr_t *prefix, uint8_t prefix_len)
+{
+    gnrc_sixlowpan_ctx_t *ctx;
+    uint8_t cid = 0;
+
+    /* find first free context ID */
+    while (((ctx = gnrc_sixlowpan_ctx_lookup_id(cid)) != NULL) &&
+           ((ctx->prefix_len != prefix_len) ||
+            (ipv6_addr_match_prefix(&ctx->prefix, prefix) < prefix_len))) {
+        cid++;
+    }
+    gnrc_sixlowpan_ctx_update(cid, (ipv6_addr_t *)prefix, prefix_len,
```

Comment:
if nothing is found, updating context 0 is fine?

Reply:
Arghs, I meant to check if it is < 16 (15 is max, so there wouldn't be any overflow anyway ;-)).

Reply:
Done

Reply:
> Done

you cannot possibly have tested this in that short amount of time! :)

Reply:
Nope, did not test, I'm just very confident in my code (but I will do ;-)).